landscape.yml

```diff
@@ -386,6 +386,11 @@ landscape:
             homepage_url: https://github.com/tencent
             logo: tencent.svg
             crunchbase: https://www.crunchbase.com/organization/tencent
+          - item:
+            name: TomTom (Member)
+            homepage_url: https://www.tomtom.com
+            logo: tomtom.svg
+            crunchbase: https://www.crunchbase.com/organization/tomtom-international
```

Comment:
To be added as TODO General Member in the Landscape, organizations must first complete the [TODO General membership application](https://enrollment.lfx.linuxfoundation.org/?project=todogroup) and meet the [criteria](https://todogroup.org/join/).

Until then, could you please remove the lines related to "Member" and only keep the ones related to "Adopter"? We can add them later by submitting a new PR 👍 🙂

Reply:
I forgot to enroll ;-) I will do that today. Once that is in order I will ping you so we can include it in this PR